spec/factories/harvest_job.rb

```diff
@@ -9,5 +9,35 @@
     throughput { 0 }
     duration { 0 }
     status { 'ready' }
+    environment { 'staging' }
+
+    harvest_failure { OpenStruct.new(backtrace: '{}') }
+    invalid_records { [] }
+    failed_records { [] }
+
+    trait :active do
+      id { 1 }
+      status { 'active' }
+    end
+
+    trait :finished do
+      id { 1 }
+      status { 'finished' }
+    end
+
+    trait :stopped do
+      id { 1 }
+      status { 'stopped' }
+    end
+
+    trait :finished do
+      id { 1 }
+      status { 'finished' }
+    end
+
+    trait :failed do
+      id { 1 }
+      status { 'failed' }
+    end
```

Comment:
The `id { 1 }` makes these traits not very useable outside of specific scenarios where no other HarvestJob's exist. If you could remove it and decouple the factory from the test that'd be a win imo :)